automated/linux/crypto/cryptsetup.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+. ../../lib/sh-test-lib
+
+OUTPUT="$(pwd)/output"
+mkdir -p "${OUTPUT}"
+RESULT_FILE="${OUTPUT}/result.txt"
```

Comment:
These file isn't used, which leads to that the '- ../../send-to-lava.sh ./output/result.txt' will not work as expected.

Reply:
RESULT_FILE is used in sh-test-lib when calling check_return() or exit_or_fail(). am i missing anything here?

Reply:
you are correct, I missed them.